simplified-app-ekirjasto/build.gradle.kts

```diff
@@ -619,7 +619,8 @@ dependencies {
     implementation(libs.truevfs.kernel.impl)
     implementation(libs.truevfs.kernel.spec)
 
-    implementation("readium:liblcp:1.0.0@aar")
+    implementation("readium:liblcp:2.1.0@aar")
```

Comment:
This should be changed so that the library is read from gradle.properties using local.properties override if present. This way it should work with the rest of the properties overrides.